pkg/cephfs/fsjournal.go

```diff
@@ -105,8 +112,10 @@ func undoVolReservation(ctx context.Context, volOptions *volumeOptions, vid volu
 // to generate the volume identifier for the reserved UUID
 func reserveVol(ctx context.Context, volOptions *volumeOptions, secret map[string]string) (*volumeIdentifier, error) {
 	var (
-		vi  util.CSIIdentifier
-		vid volumeIdentifier
+		vi        util.CSIIdentifier
+		vid       volumeIdentifier
+		imageUUID string
+		err       error
```

Comment:
let's not group different types of variables.

note: not related to this PR current code does this, we need to clean up that one as well.